### PR TITLE
Update pubspec examples/doc

### PR DESCRIPTION
**Pull request by Kelltimo**
Update double quotes to single quotes.

### CODE REVIEW

**Comment by Kelltimo** [comment]
@googlebot I signed it

**Comment by Kelltimo** [comment]
@googlebot I signed it

**Comment by Kelltimo** [comment]
@googlebot I signed it

File: src/tools/pub/pubspec.md [hunk 1]
```diff
@@ -288,8 +288,8 @@ pub supports Flutter SDK constraints under the `environment:` field, like so:
 
 {% prettify yaml %}
 environment:
-  sdk: ">=1.19.0 <2.0.0"
-  flutter: "^0.1.2"
+  sdk: '>=1.19.0 <2.0.0'
+  flutter: '^0.1.2'
```
Comment (chalin) [inline src/tools/pub/pubspec.md]: You don't need quotes in this case; i.e., it can be:

    flutter: ^0.1.2

**Comment by kwalrath** [comment]
Thanks for the contribution, @kelltimo! Did you sign the CLA using the same email address that you use for GitHub?

**Comment by Kelltimo** [comment]
Hi @kwalrath I did, but it doesn't seem like it worked. Is there a way to troubleshoot this?

**Comment by kwalrath** [comment]
When I look at your [raw patch](https://patch-diff.githubusercontent.com/raw/dart-lang/site-www/pull/865.patch), I see a @users.noreply.github.com address. Maybe you have some sort of privacy setting that obscures your email address, and our CLA bot can't figure out what your real address is. Maybe try signing the CLA with the weird email address that's in the raw patch?

**Comment by Kelltimo** [comment]
@kwalrath yes you were right. I had my email not set to public, updated and submitted a new pull request.

**Comment by kwalrath** [comment]
New PR (#875) has been merged.

**Pull request closed without merging**